src/pages/AuthPage.tsx

```diff
@@ -3,10 +3,20 @@ import { toast } from 'react-toastify';
 import { Card, Input, Button, Typography } from '@material-tailwind/react';
 import { authService } from '@services';
 import { useUserStore } from '@states';
+import { useState } from 'react';
+import { EyeIcon, EyeSlashIcon } from '@heroicons/react/24/outline';
+import { validateSchema } from '@utils';
+import { yupResolver } from '@hookform/resolvers/yup';
 
 export function AuthPage() {
```

Comment:
You can tell me if i missed something, but why AuthPage is a function declaration while AuthLayout is a function expression? You can try the arrow function, which we used in a previous project.

Reply:
Oh, i jusst use the template of tickflow. 
1. Is there any problem when using a DF into a EF?
2. Regarding to your recommends, should I convert both into arrow functions?